lumeer-core/src/main/java/io/lumeer/core/facade/LinkTypeFacade.java

```diff
@@ -19,27 +19,62 @@
 
 package io.lumeer.core.facade;
 
+import io.lumeer.api.model.Collection;
 import io.lumeer.api.model.LinkType;
-import io.lumeer.storage.api.query.SearchQuery;
+import io.lumeer.api.model.Query;
+import io.lumeer.api.model.Role;
+import io.lumeer.core.PermissionsChecker;
+import io.lumeer.storage.api.dao.CollectionDao;
+import io.lumeer.storage.api.dao.LinkTypeDao;
 
+import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import javax.inject.Inject;
 
 public class LinkTypeFacade {
 
+   @Inject
+   private LinkTypeDao linkTypeDao;
+
+   @Inject
+   private CollectionDao collectionDao;
+
+   @Inject
+   protected PermissionsChecker permissionsChecker;
```

Comment:
The same applies here as well. We can just extend `AbstractFacade` and get rid of injecting common stuff.